src/PowerShellEditorServices/Services/PowerShell/Execution/SynchronousPowerShellTask.cs

```diff
@@ -382,7 +382,7 @@ internal void MaybeAddToHistory()
 
         private void CancelDebugExecution()
         {
-            if (_pwsh.Runspace.RunspaceStateInfo.IsUsable())
+            if (!_pwsh.Runspace.RunspaceStateInfo.IsUsable())
```

Comment:
Was this just a big oops?

Reply:
Oh yeah, I meant to add a review comment pointing this out

BIG ol' oops there